Automattic/wp-calypso
# Activity Log: move from "Stats > Activity" to it's own section

This PR moved the activity log from the current location that is a tab under the activity log to the sidebar. 

:before:
![screen_shot_2018-08-28_at_2_00_38_pm](https://user-images.githubusercontent.com/115071/44750674-d0eb5700-aaca-11e8-8f57-6805c425184a.png)


:after:
![screen_shot_2018-08-28_at_1_59_29_pm](https://user-images.githubusercontent.com/115071/44750622-b44f1f00-aaca-11e8-9da0-eecf4a460fd3.png)

Everything should still work like before. 

Related: p9rlnk-u5-p2


To test:
1. Run the branch locally, or on calypso.live
2. Ensure the 'Activity' submenu no longer appears on the stats page
3. Ensure that manually going to 'stats/activity' redirects to 'activity-log'
4. There should be an 'activity' link in the side bar
4. The activity log should be visible and function as before, but now living at '/activity-log' 
5. Guided tour for activity log should function as before
